### ISSUE TITLE
Magento 2.4.4 PHP 8.1 Deprecated issue

### ISSUE THREAD

**Issue by hanhpv** [cloudinary/cloudinary_magento2#115]
Magento 2.4.4
Extension version: 1.18.0
PHP 8.1

In admin go to Catalog > Products
```
Deprecated Functionality: strlen(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/vendor/cloudinary/cloudinary_php/src/Cloudinary.php on line 546
Exception in /var/www/html/vendor/magento/framework/App/ErrorHandler.php:61
```

<img width="960" alt="image" src="https://user-images.githubusercontent.com/5069672/177709964-04a5f7e7-de52-47a7-8e9e-78fd1f4ff006.png">

**Comment by hanhpv** [comment]
Hi @wissam-khalili

Thanks for looking into this.

1. Yes it is working now on live site with Magento 2.4.2, PHP 7.4 (we're upgrading to Magento 2.4.4)
2. If I set Enable Cloudinary to No in store configuration, it is working fine.
3. Unfortunately, it is a customer site so I can't share.
4. No, mostly a default Magento site, with Cloudinary installed.
5. I do have some other 3rd party modules, but I don't think they relate. As stated in (1), it is still working fine on live site.

Maybe a PHP 8 compatible issue? I notice the latest version of the module requires `"cloudinary/cloudinary_php": "^1.20.0"`, but `cloudinary/cloudinary_php` is now `2.7.1`

**Comment by momoip** [comment]
Hi @hanhpv,
We are working on supporting PHP 8 and will let you know once we have an ETA.

Regards,
Mo

**Comment by wissam-khalili** [comment]
Hi @hanhpv,

I would like to update you that we have released a new Magento version 1.19 which supports PHP 8.

Regards,
Wissam
